src/main/java/control/ControllerManager.java

```diff
@@ -146,7 +148,7 @@ private void handleCancel(MouseEvent event) {
      *
      * @param block ShotBlock to set as active
      */
-    public void setActiveShotBlock(ShotBlock block) {
+    public void setActiveShotBlock(Optional<ShotBlock> block) {
```

Comment:
Won't work due to the fact that Optional.of() throws an exception when given null. And the point of using Optional would be to get rid of null variables anyways. So you could give it Optional.empty() instead